CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Read more details in the issue [#52: Releases and versioning](https://github.com
 
 <!-- markdownlint-configure-file {"MD024": { "siblings_only": true } } -->
 
+## [v1.3.2] Unreleased
+
+### Changed
+
+- Use the [setup-openfoam](https://github.com/marketplace/actions/setup-openfoam) action to install OpenFOAM within CI workflows [#338](https://github.com/precice/openfoam-adapter/pull/338).
```

Comment:
This line would go into a separate file according to the number of the PR `338.md` into a directory called `changelog-entries` (to be created). The file edited here is only edited throughout the release. See also https://github.com/precice/openfoam-adapter/blob/develop/CONTRIBUTING.md#changelog

Reply:
@davidscn Thanks. I'll take care of it.

Reply:
Done.